app/templates/index.html

```diff
@@ -30,10 +30,14 @@ <h3 class="mbr-header-text">Follow all your lifestyle interests in one <span cla
             </div>
             <div class="mbr-box-magnet mbr-box-magnet-top-left mbr-section-right mockup-frames col-sm-8">
               <div class="mbr-figure-container phone-container">
+                <p>Test1111 </p>
```

Comment:
I just added it to show you that not even the p comment  is being displayed in the section. Do you think that 2 different animations on both the ipad and iphone had some issues running at the same time